node.go

```diff
@@ -82,7 +83,7 @@ func (n *Node) SetMaxOpenConns(openConns int) {
 // processed by the server. Note that this guarantee only applies to queries
 // run on the given connection
 func (n *Node) NoReplyWait() error {
-	return n.pool.Exec(nil, Query{ // nil = connection opts' timeout
+	return n.pool.Exec(context.TODO(), Query{ // nil = connection opts' timeout
```

Comment:
As `context.TODO()` is a non-nil context, this line actually changes the behavior that described in the comment in the same line.

Reply:
passing a nil context is discouraged by static check, added comparison to context.TODO() in Query to behave the same.